src/components/PlaylistPage.vue

```diff
@@ -158,7 +158,7 @@ export default {
         downloadPlaylistAsTxt() {
             var data = "";
             this.playlist.relatedStreams.forEach(element => {
-                data += "https://piped.video" + element.url + "\n";
+                data += import.meta.env.VITE_PIPED_FRONTEND + element.url + "\n";
```

Comment:
I'm not so sure that this should be changed, this could break link discovery for things like `yt-dlp`, etc.

Reply:
Oh, as in tools like that won't discover URLs because they won't be piped.video any more for deployments on other domains? I suppose it's possible, but in turn it's not great when playlists send you to an entirely different domain either.

I'm not super married to this particular change, so I'll revert it if you prefer.